drivers/pinctrl/Kconfig.npcx

```diff
@@ -0,0 +1,11 @@
+# NPCX Pin Controller configuration options
+
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config PINCTRL_NPCX
+	bool "Nuvoton NPCX embedded controller (EC) pin controller driver"
+	depends on SOC_FAMILY_NPCX
```

Comment:
nit: please default based on DT compat status (check other drivers)

Reply:
Done.

Reply:
can't see the change (at least in the first commit), so not resolved.

Reply:
Oops. I referred to the wrong [Kconfig file](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/pinctrl/Kconfig.nrf#L8). Will do it later.

Reply:
Done.